packages/main/src/utils/Strings.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Escape a regular expression text so that it can be used in a regular expression.
+ * @param {string} text The raw text
+ * @returns The regular expression text, with special characters having been escaped.
+ */
+export function escapeRegex(text) {
+  return text.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+}
+
+/**
+ * Replace all instances of a particular find text.
+ * @param {string} text The text to find/replace on
+ * @param {string} find The string to find
+ * @param {string|(s:string)=>string} replace The replace string or function
+ * @param {boolean=} caseInsensitive true if the replace is case insensitive, false by default
+ * @return {string} The string with all matches replaced.
+ */
+export function replaceAll(text, find, replace, caseInsensitive) {
+  return text.replace(new RegExp(escapeRegex(find), `${caseInsensitive ? 'i' : ''}g`), replace);
+}
+
+// Cache the regular expressions for repeated use
+const markupReplace = `&=&amp,>=&gt;,<=&lt;,"=&quot;,'=&#39;`.split(",")
```

Comment:
This might be a little weird to define the replacements, I thought it might compress a little better like this.